plugins/automation/plugin.rb

```diff
@@ -39,21 +39,9 @@ def self.get_active_automation
 end
 
 require_relative "lib/discourse_automation/engine"
-require_relative "lib/discourse_automation/scriptable"
-require_relative "lib/discourse_automation/triggerable"
-require_relative "lib/plugin/instance"
 
 after_initialize do
   %w[
-    app/jobs/regular/discourse_automation_call_zapier_webhook
-    app/jobs/regular/discourse_automation_trigger
-    app/jobs/scheduled/discourse_automation_tracker
-    app/jobs/scheduled/stalled_topic_tracker
-    app/jobs/scheduled/stalled_wiki_tracker
-    app/queries/stalled_topic_finder
-    app/services/discourse_automation/user_badge_granted_handler
-    lib/discourse_automation/event_handlers
-    lib/discourse_automation/post_extension
```

Comment:
Doing this means the scheduled jobs won't be autoloaded in dev anymore, look what we did in chat (and other places) recently:

https://github.com/discourse/discourse/commit/e949684fc54e629035b8883dabd9ef2916251da1

Reply:
Or even better:

https://github.com/discourse/discourse/pull/26853/files

Reply:
done!